pkg/cli/admin/upgrade/status/mco/layered_node_state.go

```diff
@@ -19,8 +20,10 @@ import (
 // status.go with this code, then repackage this so that it can be used by any
 // portion of the MCO which needs to interrogate or mutate node state.
 type LayeredNodeState struct {
-	node                   *corev1.Node
-	ReasonOfUnavailability string
+	node                               *corev1.Node
+	ReasonOfUnavailability             string
+	MessageOfUnavailability            string
+	LastTransitionTimeOfUnavailability time.Time
```

Comment:
reason+message+last transition time is essentially a `Condition` where the `...OfUnavailability` takes the role of the `Available` type and assumes `Status=False`.

I'd like to get to the state where we do not hide unavailability reasons when there are more of them (see comment below), for which we'd need to do something like this

```
type badCondition struct {
   reason string
   message string
   lastTransitionTime time.Time
}

type  LayeredNodeState struct {
    node *corev1.Node
    unavailable []badCondition
}
```

Node is unavailable if `unavailable` is non empty. Compute the final message as a `|`-join of the individual reasons (`Node is not ready | Node has disk pressure` or potentially `Not ready | Disk pressure`, see below).